microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Manage PIM policies using Microsoft Graph PowerShell"
+description: "Learn how to manage PIM policies in Microsoft Graph PowerShell"
+
+ms.topic: how-to
+ms.date: 09/12/2022
+author: msewaweru
+manager: mwongerapk
+ms.author: eunicewaweru
+---
+
+# Manage PIM policies in Microsoft Graph PowerShell
+
+The Azure AD Privileged Identity Management (PIM) service allows role administrators to make time-bound admin role assignments. Each role has a set of settings that manage it. For example, activation and assignment settings. These role settings are called policies. Each policy has a set of rules.
+
+In this how-to guide, you'll list and update these policy rules.
+
+## Prerequisites
+
+To successfully complete this guide, make sure you have the required prerequisites:
+
+1. Microsoft Graph PowerShell SDK is installed. Follow the [Install the Microsoft Graph PowerShell SDK](installation.md) guide to install the SDK.  
+1. To use the Azure AD Privileged Identity Management, you must have one of the following licenses:
+    - Azure AD Premium P2
+    - Enterprise Mobility + Security (EMS) E5 license
+1. Microsoft Graph PowerShell using a global administrator role and the appropriate permissions. For this tutorial, the `RoleManagement.ReadWrite.Directory` delegated permission is required. To set the permissions in Microsoft Graph PowerShell, run;
+
+    ```powershell
+    Connect-MgGraph -Scopes "RoleManagement.ReadWrite.Directory"
+    ```
+
+    Select **Consent on behalf of your organization** before accepting in the sign-in dialog box.
+
+## Step 1: List policies
+
+### List all policies
+
+To list all policies at directory level, run:
+
+```powershell
+Get-MgPolicyRoleManagementPolicy -Filter "scopeId eq '/' and scopeType eq 'DirectoryRole'"
```

Comment:
In Step 2, I notice we define what the cmdlet returns then give the snippet. Should we do the same for step 1?

Reply:
I have removed the cmdlet definition and instead defined what assignments are. Thanks.